app/routes/endemics/check-answers.js

```diff
@@ -446,24 +447,22 @@ const getHandler = {
                 },
                 value: {
                   html:
-                    typeof sheepTest.result === 'object'
-                      ? sheepTest.result
-                          .map(
-                            (testResult) =>
-                              `${testResult.diseaseType} (${testResult.testResult})</br>`
-                          )
-                          .join(' ')
-                      : `${
-                          sheepTestTypes[
-                            sessionData?.sheepEndemicsPackage
-                          ].find((test) => test.value === sheepTest.diseaseType)
-                            .text
-                        } (${
-                          sheepTestResultsType[sheepTest.diseaseType].find(
-                            (resultType) =>
-                              resultType.value === sheepTest.result
-                          ).text
-                        })`
+                  typeof sheepTest.result === 'object'
+                    ? sheepTest.result
+                        .map(
+                          (testResult) =>
+                          `${testResult.diseaseType} (${testResult.testResult})</br>`
+                        )
+                        .join(' ')
+                    : `${sheepTestTypes[
+                      sessionData?.sheepEndemicsPackage
+                    ].find((test) => test.value === sheepTest.diseaseType)
+                      .text
+                    } (${sheepTestResultsType[sheepTest.diseaseType].find(
+                      (resultType) =>
+                        resultType.value === sheepTest.result
+                    ).text
+                    })`
```

Comment:
Not your work but this is pretty messy. Any chance we can flatten it out a bit?

Reply:
Sorry I meant more the logic than the formatting :). But never mind we can loop back to this, it's not really part of this PR.